packages/suite/src/views/dashboard/AssetsView/AssetCard/AssetCard.tsx

```diff
@@ -87,13 +87,13 @@ const FailedContainer = styled.div`
     }
 `;
 
-interface AssetCardProps {
+type AssetCardProps = {
```

Comment:
I'm glad that I'm not the only one with a preference for `type` over `interface`  :slightly_smiling_face: 
(ofc except cases where the specific features of `interface` are needed)